### PR TITLE
Travis sharded test fixes

### PR DESCRIPTION
**Pull request by snopoke**
@millerdev

### CODE REVIEW

File: .travis/localsettings.py [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 from docker.dockersettings import *
 
+TEST_RUNNER = 'testrunner.TwoStageTestRunner'
```
Comment (snopoke) [inline .travis/localsettings.py]: After your fix this was the main culprit

**Comment by snopoke** [comment]
:white_check_mark:

File: .travis/test_runner.sh [hunk 2]
```diff
@@ -10,13 +10,15 @@
...
 
-TESTS="$1"
+TESTS="$@"
 COMMAND="coverage run manage.py test --noinput --failfast --traceback --verbosity=2"
 
 /moto-s3/env/bin/moto_server s3 &
 
 if [ -z ${COMMAND_OVERRIDE} ]; then
+    echo "Running tests: $TESTS"
     $COMMAND $TESTS
```
Comment (millerdev) [inline .travis/test_runner.sh]: This is broken:

test.sh
```sh
#!/bin/bash
echo "sh args: $@"
TESTS="$@"
python ./test.py $TESTS
python ./test.py "$TESTS"
python ./test.py "$@"
```
test.py
```python
#! /usr/bin/env python
import sys
print(sys.argv)
```

output:
```sh
$ chmod +x test.sh
$ ./test.sh "T W O" "args"
sh args: T W O args
['./test.py', 'T', 'W', 'O', 'args']
['./test.py', 'T W O args']
['./test.py', 'T W O', 'args']

Comment (czue) [inline .travis/test_runner.sh]: sorry @millerdev I probably shouldn't have merged without context. feel free to revert if you want

Comment (millerdev) [inline .travis/test_runner.sh]: bash is a terrible language. extremely hard to understand. we should avoid it at all costs.

Comment (millerdev) [inline .travis/test_runner.sh]: fixed in https://github.com/dimagi/commcare-hq/pull/11568/files#diff-69316794bdf7ed7ed1be89f32cd69da0R19

**Comment by snopoke** [comment]
Applogies, should have put a 'open for review don't merge' label on this.

File: .travis/test_runner.sh (hunk 2 above)
Comment (snopoke) [inline .travis/test_runner.sh]: Thanks for the fix though in this case calling with those arguments would be an error anyway

**Pull request merged**